lib/facter/falco.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+Facter.add(:falco_driver_version) do
+  confine kernel: :Linux
```

Comment:
should we instead confine to  Facter::Util::Resolution.which('falco')?

Reply:
Yeah that would make sense for the fact as it stands.

In the case where `falco::package_ensure => 'latest'` and there is a newer falco to be installed, the puppet fact would cause the exec to only trigger on the second run after the update, so that would mean it is a two-puppet-run solution.

I think the only way around this would be a type & provider instead of an `exec` resource to manage the kernel module compilation, but I think this solution could work for now until someone has time to rewrite the exec as a custom resource.